engine/src/main/java/com/redhat/cloud/notifications/templates/DrawerTemplateMigrationService.java

```diff
@@ -0,0 +1,362 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.cronutils.utils.StringUtils;
+import com.redhat.cloud.notifications.models.EventType;
+import com.redhat.cloud.notifications.models.IntegrationTemplate;
+import com.redhat.cloud.notifications.models.Template;
+import io.quarkus.logging.Log;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
+import javax.transaction.Transactional;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static com.redhat.cloud.notifications.events.ConnectorReceiver.INTEGRATION_FAILED_EVENT_TYPE;
+import static com.redhat.cloud.notifications.events.IntegrationDisabledNotifier.INTEGRATION_DISABLED_EVENT_TYPE;
+import static com.redhat.cloud.notifications.processors.email.aggregators.AdvisorEmailAggregator.*;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@ApplicationScoped
+public class DrawerTemplateMigrationService {
+
+    @Inject
+    EntityManager entityManager;
+
+    @Transactional
+    public List<String> migrate() {
+        List<String> warnings = new ArrayList<>();
+
+        Log.debug("Migration starting");
+
+        // Advisor
+        createDrawerIntegrationTemplate(
+            warnings, "rhel", "advisor", List.of(DEACTIVATED_RECOMMENDATION),
+            "Advisor/deactivatedRecommendationBody", "html", "Advisor deactivated recommendation drawer body"
+        );
+        createDrawerIntegrationTemplate(
+            warnings, "rhel", "advisor", List.of(NEW_RECOMMENDATION),
+            "Advisor/newRecommendationBody", "html", "Advisor new recommendation drawer body"
```

Comment:
I wonder if we should have a clearer naming strategy for all DB templates (not only the drawer ones).

For example, instead of having:
- `Advisor/newRecommendationInstantEmailBody`
- `Advisor/newRecommendationBody`

we could use:
- `Advisor/email/newRecommendationBody`
- `Advisor/drawer/newRecommendationBody`

Changing the templates names wouldn't require to move the files. They would stay where they are right now.

WDYT?